fapolicy_analyzer/ui/profiler_page.py

```diff
@@ -73,9 +85,26 @@ def __init__(self, fapd_manager):
     def analyze_button_sensitivity(self):
         return self.analysis_available
 
+    def on_next_system(self, system):
+        profilerDict = system["profiler"]
+        self.update_field_text(profilerDict)
+
+    def update_field_text(self, profilerDict):
+        prefixes = ["execute", "arg", "user", "dir", "env"]
+        if not profilerDict:
+            profilerDict = {prefix + "Text": "" for prefix in prefixes}
```

Comment:
If you set the default value in the `profiler_reducer` to the dict you need as a default you need, you wouldn't need to build one there. On line 35 of `profiler_reducer.py` change `{}` to `{<all the keys you need with values of empty string>}`.